### ISSUE TITLE
【EESS】New submissions for Fri, 21 Jul 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#448]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Is Grad-CAM Explainable in Medical Images?
 - **Authors:** Subhashis Suara, Aayush Jha, Pratik Sinha, Arif Ahmed Sekh
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Computers and Society (cs.CY)
 - **Arxiv link:** https://arxiv.org/abs/2307.10506
 - **Pdf link:** https://arxiv.org/pdf/2307.10506
 - **Abstract**
 Explainable Deep Learning has gained significant attention in the field of artificial intelligence (AI), particularly in domains such as medical imaging, where accurate and interpretable machine learning models are crucial for effective diagnosis and treatment planning. Grad-CAM is a baseline that highlights the most critical regions of an image used in a deep learning model's decision-making process, increasing interpretability and trust in the results. It is applied in many computer vision (CV) tasks such as classification and explanation. This study explores the principles of Explainable Deep Learning and its relevance to medical imaging, discusses various explainability techniques and their limitations, and examines medical imaging applications of Grad-CAM. The findings highlight the potential of Explainable Deep Learning and Grad-CAM in improving the accuracy and interpretability of deep learning models in medical imaging. The code is available in (will be available).
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Transfer Learning and Bias Correction with Pre-trained Audio Embeddings
 - **Authors:** Changhong Wang, Gaël Richard, Brian McFee
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2307.10834
 - **Pdf link:** https://arxiv.org/pdf/2307.10834
 - **Abstract**
 Deep neural network models have become the dominant approach to a large variety of tasks within music information retrieval (MIR). These models generally require large amounts of (annotated) training data to achieve high accuracy. Because not all applications in MIR have sufficient quantities of training data, it is becoming increasingly common to transfer models across domains. This approach allows representations derived for one task to be applied to another, and can result in high accuracy with less stringent training data requirements for the downstream task. However, the properties of pre-trained audio embeddings are not fully understood. Specifically, and unlike traditionally engineered features, the representations extracted from pre-trained deep networks may embed and propagate biases from the model's training regime. This work investigates the phenomenon of bias propagation in the context of pre-trained audio representations for the task of instrument recognition. We first demonstrate that three different pre-trained representations (VGGish, OpenL3, and YAMNet) exhibit comparable performance when constrained to a single dataset, but differ in their ability to generalize across datasets (OpenMIC and IRMAS). We then investigate dataset identity and genre distribution as potential sources of bias. Finally, we propose and evaluate post-processing countermeasures to mitigate the effects of bias, and improve generalization across datasets.
## Keyword: video retrieval
There is no result 
### Efficient Unified Demosaicing for Bayer and Non-Bayer Patterned Image  Sensors
 - **Authors:** Haechang Lee, Dongwon Park, Wongi Jeong, Kijeong Kim, Hyunwoo Je, Dongil Ryu, Se Young Chun
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2307.10667
 - **Pdf link:** https://arxiv.org/pdf/2307.10667
 - **Abstract**
 As the physical size of recent CMOS image sensors (CIS) gets smaller, the latest mobile cameras are adopting unique non-Bayer color filter array (CFA) patterns (e.g., Quad, Nona, QxQ), which consist of homogeneous color units with adjacent pixels. These non-Bayer sensors are superior to conventional Bayer CFA thanks to their changeable pixel-bin sizes for different light conditions but may introduce visual artifacts during demosaicing due to their inherent pixel pattern structures and sensor hardware characteristics. Previous demosaicing methods have primarily focused on Bayer CFA, necessitating distinct reconstruction methods for non-Bayer patterned CIS with various CFA modes under different lighting conditions. In this work, we propose an efficient unified demosaicing method that can be applied to both conventional Bayer RAW and various non-Bayer CFAs' RAW data in different operation modes. Our Knowledge Learning-based demosaicing model for Adaptive Patterns, namely KLAP, utilizes CFA-adaptive filters for only 1% key filters in the network for each CFA, but still manages to effectively demosaic all the CFAs, yielding comparable performance to the large-scale models. Furthermore, by employing meta-learning during inference (KLAP-M), our model is able to eliminate unknown sensor-generic artifacts in real RAW data, effectively bridging the gap between synthetic images and real sensor RAW. Our KLAP and KLAP-M methods achieved state-of-the-art demosaicing performance in both synthetic and real RAW data of Bayer and non-Bayer CFAs.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Individualization of atrial tachycardia models for clinical  applications: Performance of fiber-independent model
 - **Authors:** Jiyue He, Arkady Pertsov, John Bullinga, Rahul Mangharam
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2307.10592
 - **Pdf link:** https://arxiv.org/pdf/2307.10592
 - **Abstract**
 One of the challenges in the development of patient-specific models of cardiac arrhythmias for clinical applications has been accounting for myocardial fiber organization. The fiber varies significantly from heart to heart, but cannot be directly measured in live tissue. The goal of this paper is to evaluate in-silico the accuracy of left atrium activation maps produced by a fiber-independent (isotropic) model with tuned diffusion coefficients, compares to a model incorporating myocardial fibers with the same geometry. For this study we utilize publicly available DT-MRI data from 7 ex-vivo hearts. The comparison is carried out in 51 cases of focal and rotor arrhythmias located in different regions of the atria. On average, the local activation time accuracy is 96% for focal and 93% for rotor arrhythmias. Given its reasonably good performance and the availability of readily accessible data for model tuning in cardiac ablation procedures, the fiber-independent model could be a promising tool for clinical applications.
### A Novel Unsupervised Graph Wavelet Autoencoder for Mechanical System  Fault Detection
 - **Authors:** Tianfu Li, Chuang Suna, Ruqiang Yan, Xuefeng Chen, Olga Fink
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.10676
 - **Pdf link:** https://arxiv.org/pdf/2307.10676
 - **Abstract**
 Reliable fault detection is an essential requirement for safe and efficient operation of complex mechanical systems in various industrial applications. Despite the abundance of existing approaches and the maturity of the fault detection research field, the interdependencies between condition monitoring data have often been overlooked. Recently, graph neural networks have been proposed as a solution for learning the interdependencies among data, and the graph autoencoder (GAE) architecture, similar to standard autoencoders, has gained widespread use in fault detection. However, both the GAE and the graph variational autoencoder (GVAE) have fixed receptive fields, limiting their ability to extract multiscale features and model performance. To overcome these limitations, we propose two graph neural network models: the graph wavelet autoencoder (GWAE), and the graph wavelet variational autoencoder (GWVAE). GWAE consists mainly of the spectral graph wavelet convolutional (SGWConv) encoder and a feature decoder, while GWVAE is the variational form of GWAE. The developed SGWConv is built upon the spectral graph wavelet transform which can realize multiscale feature extraction by decomposing the graph signal into one scaling function coefficient and several spectral graph wavelet coefficients. To achieve unsupervised mechanical system fault detection, we transform the collected system signals into PathGraph by considering the neighboring relationships of each data sample. Fault detection is then achieved by evaluating the reconstruction errors of normal and abnormal samples. We carried out experiments on two condition monitoring datasets collected from fuel control systems and one acoustic monitoring dataset from a valve. The results show that the proposed methods improve the performance by around 3%~4% compared to the comparison methods.
### Confidence intervals for performance estimates in 3D medical image  segmentation
 - **Authors:** R. El Jurdi, G. Varoquax, O. Colliot
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2307.10926
 - **Pdf link:** https://arxiv.org/pdf/2307.10926
 - **Abstract**
 Medical segmentation models are evaluated empirically. As such an evaluation is based on a limited set of example images, it is unavoidably noisy. Beyond a mean performance measure, reporting confidence intervals is thus crucial. However, this is rarely done in medical image segmentation. The width of the confidence interval depends on the test set size and on the spread of the performance measure (its standard-deviation across of the test set). For classification, many test images are needed to avoid wide confidence intervals. Segmentation, however, has not been studied, and it differs by the amount of information brought by a given test image. In this paper, we study the typical confidence intervals in medical image segmentation. We carry experiments on 3D image segmentation using the standard nnU-net framework, two datasets from the Medical Decathlon challenge and two performance measures: the Dice accuracy and the Hausdorff distance. We show that the parametric confidence intervals are reasonable approximations of the bootstrap estimates for varying test set sizes and spread of the performance metric. Importantly, we show that the test size needed to achieve a given precision is often much lower than for classification tasks. Typically, a 1% wide confidence interval requires about 100-200 test samples when the spread is low (standard-deviation around 3%). More difficult segmentation tasks may lead to higher spreads and require over 1000 samples.
